src/components/atoms/SortDropDown/SortDropDown.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import {
+  Dropdown as ReactBootstrapDropdown,
+  DropdownButton,
+} from "react-bootstrap";
+
+import { SortingOptions } from "utils/venue";
+
+import "./SortDropDown.scss";
+
+interface SpaceSortDropDownProps {
+  onClick?: (value: SortingOptions) => void;
+}
+
+export const SortDropDown: React.FC<SpaceSortDropDownProps> = ({ onClick }) => (
```

Comment:
I think this component is misnamed (or too specific in it's parameters).

As is, it's more of a `SpaceSortDropDown`. I think either:
 * rename it to `SpaceSortDropDown`
 * or, make it accept a more generic "sorting options" parameter

I don't have a strong opinion on which.

Reply:
Heh, that was the first name I gave it `SpaceSortDropDown`.
Then I noticed it was using this `VenueSortingOptions` enum and I thought "OK, I will rename that to `SpaceSortingOptions`...
And I noticed it is basically a general sort: `A-Z`, `Z-A`, `Oldest First`, `Newest First`... could basically be used for anything.

Hence the removal of the `Space` and `Venue` from the names

Reply:
Removed the `Space` from `SpaceSortDropDownProps` and moved the `title` as optional prop @ b7425bd8196ff5942d4c257358cf401de6df8750 thus making it a general sorter drop down.